src/engraving/libmscore/segment.cpp

```diff
@@ -2610,8 +2610,8 @@ qreal Segment::minHorizontalDistance(Segment* ns, bool systemHeaderGap) const
 //------------------------------------------------------
 Fraction Segment::shortestChordRest() const
 {
-    Fraction shortest = Fraction::max(); // Initializing at arbitrary high value
-    Fraction cur = Fraction::max();
+    Fraction shortest = measure()->timesig(); // Initializing at the highest possible value ( = time signature of the measure)
```

Comment:
`measure->timesig()` IMHO returns the nominal time sig of a given measure, but I guess we'd want the actual duration  here, so `measure()->ticks()`, don#t we?

Reply:
good point, thanks!